### DISCUSSION TITLE
明细表支持行列转置显示吗

### DISCUSSION THREAD

**Discussion by KzqKzq** [antvis/S2#3007] [Q&A]
@lijinke666  大佬，请问下明细表支持行列转置显示吗，之前项目一直在用s2， 现在有行列转置的这种需求，我看了下文档好像没找到相关功能，类似下面这种效果
![image](https://github.com/user-attachments/assets/ff962d23-883d-46a7-bcfa-f402f68d3d9b)

**Comment by lijinke666** [comment]
不支持, 文档没写就是不支持, 欢迎参与共建!
